lib/utils/custom-chars.js

```diff
@@ -5,7 +5,7 @@ export default class {
     this.hapChar = api.hap.Characteristic;
     this.uuids = {
       maxSpeed: 'E963F001-079E-48FF-8F27-9C2605A29F52',
-      customArea: 'E963F002-079E-48FF-8F27-9C2605A29F52',
+      predefinedArea: 'E963F002-079E-48FF-8F27-9C2605A29F52',
```

Comment:
as we spoke about, please check to see if this name change is backward compatible

Reply:
I will check this later today!

Reply:
Checking this I found (and fixed) an error re: the condition whether or not to add/remove the predefined area characteristic: so far it was removed when there was not defined at least one predefined area not taking the old custom areas into consideration too...

That being fixed: the renaming does not take effect and thus does not break the configured scenes until the characteristic is removed and added again (which is not the case during a regular update). The scenes still work although the characteristic has been renamed in the background - this is probably due to the fact that the UUID did not change. 

So, the renaming is backwards compatible from my point of view.